frontend/src/components/AboutUsSection/AboutUs.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Button } from '../shared/button';
+import ProgressBar from '../ProgressBar/ProgressBar';
+import './style.css';
+
+function AboutUs() {
+  const progressBarData = [
```

Comment:
bring the progress bar data from the backend and use useEffect

Reply:
will work for it